src/shared/types/web3.types.ts

```diff
@@ -15,3 +17,9 @@ export enum EChainId {
   POLYGON_MAINNET = 137,
   AVAX_MAINNET = 43114
 }
+
+export interface IConnectorMetadata {
```

Comment:
Wouldn't really call it a web3.type since it's really just metadata basically.
We can maybe create metadata.types.ts and add this as well as
https://github.com/MVPWorkshop/tesseract-app/blob/15f074e27114fa6d265947746fd645e9d628f27c/src/shared/constants/web3.constants.ts#L78